examples/README.md

```diff
@@ -861,6 +860,52 @@ credentials for the created cluster_ and _submit a job calling `nvidia_smi`_.
 [ml-gke.yaml]: ../community/examples/ml-gke.yaml
 [`kubernetes-operations`]: ../community/modules/scripts/kubernetes-operations/README.md
 
+### [ml-cluster.yaml] ![community-badge] ![experimental-badge]
+
+This blueprint provisions an HPC cluster running the Slurm scheduler with the
+machine learning frameworks [PyTorch] and [TensorFlow] pre-installed on every
+VM. The cluster has 2 partitions:
+
+* [A2 family VMs][a2] with the NVIDIA A100 GPU accelerator
+* [G2 family VMs][g2] with the NVIDIA L4 GPU accelerator
+
+[a2]: https://cloud.google.com/compute/docs/gpus#a100-gpus
+[g2]: https://cloud.google.com/compute/docs/gpus#l4-gpus
+
+To provision the cluster, please run:
+
+```text
+./ghpc create community/examples/ml-cluster.yaml --vars "project_id=${GOOGLE_CLOUD_PROJECT}"
+./ghpc deploy ml-example"
+```
+
+After accessing the login node, you can activate the conda environment for each
+library with:
+
+```shell
+source /etc/profile.d/conda.sh
+# to activate PyTorch
+conda activate pytorch
+# to activate TensorFlow
+conda activate tf
+```
+
+An example benchmarking job for PyTorch can be run under Slurm:
+
+```shell
+cp /var/tmp/torch_test.* .
+sbatch -N 1 torch_test.sh
+```
+
+When you are done, clean up the resources in reverse order of creation:
+
+```text
+terraform -chdir=image-builder-001/cluster destroy --auto-approve
+terraform -chdir=image-builder-001/primary destroy --auto-approve
```

Comment:
how do they destroy the image? (gcloud?)

Reply:
Flexible instructions for deleting the image (which is uniquely named) via the console are provided in the lines directly below this section of text.

Reply:
I can't seem to find it... what am I missing?

Reply:
Oh, you are right. It appears in the (similar) image-builder example but not here. Will fix.

Reply:
See a6d72f9cf048bf36a4e8a0ff49dd085492754754